umm-spec-lib/test/cmr/umm_spec/test/umm_g/granule.clj

```diff
@@ -87,6 +86,26 @@
                                :end-coordinate-1 -50
                                :start-coordinate-2 50
                                :end-coordinate-2 100})
+    :product-specific-attributes [(umm-lib-g/map->ProductSpecificAttributeRef
+                                    {:name "AdditionalAttribute1 Name1"
```

Comment:
Additional attributes can be a bunch of different types - float, integer, booleans that get treated as strings, etc. It looks like we're only testing with strings. Shouldn't we test the other types as well?

Reply:
This is Additional Attributes for granules. It is much easier than collections. Thank God!

Reply:
Since this is just for the translation part (validation of types should already be handled separately), I'm ok with merging this in.

I'd recommend changing the examples to:
```{:name "EVI1KM16DAYQCLASSPERCENTAGE" :values ["Value5" "Value6"]}```
rather than repeating the name within the values, but we can discuss that with Erich when he comes back.